portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -40,7 +52,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String newComment = request.getParameter("new-comment");
     if (!newComment.isEmpty()){
-      this.comments.add(newComment);
+      Entity commentEntity = new Entity("Comment");
+      commentEntity.setProperty("content", newComment);
```

Comment:
I recommend you also add a timestamp property similar to the examples. This way you can show the comments in order . Typically when you read from a database, order is not guaranteed so its a good idea to store time as well.

Reply:
Oh I didn't know that. Thanks for mentioning. A timestamp has been added to each comment.